packages/blocks/src/lib/navigation-map/components/nav/index.svelte

```diff
@@ -12,36 +11,52 @@ const dispatch = createEventDispatcher<{
   'update-obstacles': Obstacle[]
 }>();
 
-const handleTabSelect = (event: CustomEvent<{ value: string }>) => {
-  $tab = event.detail.value as NavigationTabType;
-};
-
 const handleUpdateObstacle = (event: CustomEvent<Obstacle[]>) => {
   dispatch('update-obstacles', event.detail)
 }
 
 </script>
 
 <nav class='sm:h-full sm:w-[350px]'>
-  <Tabs
-    tabs={$tabs}
-    selected={$tab}
-    on:input={handleTabSelect}
-  />
+  <ol class='ml-4 mt-4 mb-2 flex flex-wrap items-center'>
+    {#each $tabs as tabTitle}
+      {@const selected = $tab === tabTitle}
+      <li>
+        <button
+          class="border-b px-4 py-1 capitalize text-sm tracking-normal"
+          class:border-black={selected}
+          class:font-bold={selected}
+          class:text-gray-600={!selected}
+          on:click={() => { $tab = tabTitle }}
+        >
+          {tabTitle}
+        </button>
+      </li>
+    {/each}
+  </ol>
 
-  <ul
-    on:mouseleave={() => ($hovered = null)}
-    class='px-4 py-2 sm:h-[calc(100%-30px)] overflow-y-scroll'
-  >
-    {#if $tab === NavigationTab.Waypoints}
+  {#if $tab === NavigationTab.Waypoints}
+    <ul
+      on:mouseleave={() => ($hovered = null)}
```

Comment:
Is this another way of doing `hovered.set(null)`? If so, I kinda prefer using `.set` explicitly. It's more portable to vanilla TS, and I think it does a better job of reminding future readers that `hovered` is a box, not a value

Reply:
I'm fine with both `$` and the vanilla syntax, although to me `$hovered = null` is a very transparent sugar. The reason I was sticking with `$` is consistency with the read ops above, which I wouldn't want to change since I think it's much more syntactically elegant than `.subscribe()`. Can change to `.set()` though if you or other reviewers are more comfortable with it